src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -162,18 +162,27 @@ class ArchiveExplorer extends React.Component {
      */
     sortItemList(itemList) {
         const { sortBy, sortDirection } = this.state;
+        let sortedItems;
 
         if (!sortBy.length) {
             return itemList;
         }
 
-        const sortedItems = itemList.sort((a, b) => {
-            if (typeof a[sortBy] === 'number' && typeof b[sortBy] === 'number') {
-                return a[sortBy] - b[sortBy];
-            }
+        try {
+            sortedItems = itemList.sort((a, b) => {
+                if (!a[sortBy] || !b[sortBy]) {
```

Comment:
Should we also check `!a || !b` for null/undefined values?

Reply:
I think the keys are always there, but the values maybe be `null | value`, but I can check without much effort 👍